src/utils/__tests__/lodash-test.tsx

```diff
@@ -0,0 +1,115 @@
+import {debounce, isEqual} from '../lodash';
+
+beforeEach(() => {
+    jest.useFakeTimers();
+});
+
+describe('debounce', () => {
```

Comment:
why describe?

Reply:
It is a way to organize the test suite. All `debounce` tests are placed inside the `debounce` describe section, same with `isEqual` or any other method.

According to jest doc:
> `describe()` creates a block that groups together several related tests

And when you run the tests, they are shown with a pretty hierarchy
<img width="382" alt="image" src="https://github.com/Telefonica/mistica-web/assets/579417/f0edeafe-c605-42d2-8a77-2869fc935b1c">

Reply:
I don't like it, and we don't use it anywhere. Organice the tests by file should be more than enough. And you already have a "debounce" or "isEqual" prefix in all your test names. It adds an extra nesting and can be tricky when combined with before/after methods